src/cf-vars.less

```diff
@@ -44,15 +47,18 @@
     - cf-core
 */
 
-// Sizing variables 
+// Sizing variables
 
 @base-font-size-px:             16px;
 @base-line-height-px:           22px;
 @base-line-height:              unit(@base-line-height-px / @base-font-size-px);
 @mobile-max:                    599px;
 @tablet-min:                    600px;
+@tablet-max:                    800px;
+@desktop-min:                   801px;
+@wall-min:                      1200px;
```

Comment:
I know we have some terminology discussion outstanding, but if we're going ahead with this change now, can I quibble with the term "wall"?

I have an idea of what is meant by it, but I'm not sure. So I guess my quibble is, I don't know what is meant by it.